agent/health_endpoint.go

```diff
@@ -273,6 +277,27 @@ func (s *HTTPServer) healthServiceNodes(resp http.ResponseWriter, req *http.Requ
 	return out.Nodes, nil
 }
 
+func getBoolQueryParam(params url.Values, key string) (bool, error) {
+	var param bool
+	if _, ok := params[key]; ok {
+		val := params.Get(key)
+		// Orginally a comment declared this check should be removed after Consul
+		// 0.10, to no longer support using ?passing without a value. However, I
+		// think this is a reasonable experience for a user and so am keeping it
+		// here.
```

Comment:
Thoughts on this are welcome. In my opinion, allowing just `?ingress` as well as `?ingress=true` is a good UX.

Reply:
Agreed, allowing both is what we do in other places like `?recurse` in the KV path.